shapedetector.py

```diff
@@ -66,7 +69,7 @@ def detect_shapes():
         M = cv2.moments(c)
 
         # ignore contours that are too small to be species
-        if c.shape[0] > 2 and area/(resized.shape[0]*resized.shape[1]) > 0.002:
+        if c.shape[0] > 2 and area/(resized.shape[0]*resized.shape[1]) > 0.002 and area/(resized.shape[0]*resized.shape[1]) < 0.25:
```

Comment:
again, no magic numbers please.

also, you might want to put this on multiple lines:
```suggestion
        if c.shape[0] > 2 
           and area/(resized.shape[0]*resized.shape[1]) > 0.002 
           and area/(resized.shape[0]*resized.shape[1]) < 0.25:
```